projects/skia/build.sh

```diff
@@ -222,3 +222,6 @@ cp ./svg_dom_seed_corpus.zip $OUT/svg_dom_seed_corpus.zip
 
 cp out/Fuzz/api_svg_canvas $OUT/api_svg_canvas
 cp ./canvas_seed_corpus.zip $OUT/api_svg_canvas_corpus.zip
+
+cp out/Fuzz/skruntimeeffect $OUT/skruntimeeffect
+cp ./sksl_with_256_padding_seed_corpus.zip $OUT/sksl_with_256_padding_seed_corpus.zip
```

Comment:
This will not work - the name must match the name of the fuzzer. skruntimeeffect_seed_corpus.zip

Reply:
That is, the name in the $OUT directory must match.

Reply:
I have fixed issue and build and run the code, please take another look.